src/openforms/scss/components/admin/_select.scss

```
@@ -0,0 +1,3 @@
+select.form-variable-dropdown {
+  max-inline-size: 300px !important;
+}
```

1. CSS selectors should only target class names, not elements or IDs
2. why the `!important`? I was trying it out in storybook and don't see a difference with or without it

```suggestion
.form-variable-dropdown {
  max-inline-size: 300px;
}
```

---

For me it wasn't working without the `!important`. I will update the css file, according to your suggestion,  and try it once more

---

I was looking at storybook - maybe there's something else in the admin that's interfering :thinking:

---

Isn't the parent class that is present in the admin `dsl-editor__node`?As it appears in the screenshot?